examples/image_retraining/retrain.py

```diff
@@ -1342,5 +1339,11 @@ def main(_):
       default='INFO',
       choices=['DEBUG', 'INFO', 'WARN', 'ERROR', 'FATAL'],
       help='How much logging output should be produced.')
+  parser.add_argument(
+      '--checkpoints_dir',
```

Comment:
The way this is used by tf.Saver() is *not* as a directory. It is used as a prefix for the several files that make up one checkpoint, and several runs with the same value of this flag will overwrite each other's data. My last run of the tool left me with:

/tmp/_retrain_checkpoint.data-00000-of-00001
/tmp/_retrain_checkpoint.index
/tmp/_retrain_checkpoint.meta

tf.Saver uses the term "path" for this  combination of directory and basename. I suggest to follow suit here and call the flag --checkpoint_path. The description should make clear this is for one checkpoint (not multiple ones).

Reply:
sorry for reply lately，i see the commit by @skvrahul，thanks all